.travis.yml

```diff
@@ -27,13 +27,13 @@ matrix:
       env: TARGET="Documentation" ENCRYPTION_LABEL="c06a05533acc"
     # Cross-compiled targets
     - os: linux
-      env: TARGET="Android-ARM" LLDB_TESTS="all" CLANG="0" COVERAGE="1" PLATFORM="1"
+      env: TARGET="Android-ARM" LLDB_TESTS="all" PLATFORM="1" COVERAGE="1"
```

Comment:
I'm wondering if we should make `PLATFORM` implicit for android testing, since it's always required. Or at least explicitly fail if you don't use it in the testing script.